File: js/tests/sumOfTwoNumbers.test.js
const addition = require("../sumofTwoNumbers");

const data = [
  [2, 3, 5],
  [-3, -6, -9],
  [7, 3, 10],
  [88, 2, 90]
];

test.each(data)("Add %i + %i", (a, b, expected) => {
  var actual = addition(a, b);

  expect(actual).toBe(expected);
});

File: CSharp/Tests/SumOfTwoNumbersTest.cs
using Xunit;

namespace CSharp.Tests
{
    public class SumOfTwoNumbersTest
    {
        [Theory]
        [InlineData(2, 3, 5)]
        [InlineData(-3, -6, -9)]
        [InlineData(7, 3, 10)]
        public void Sum_TwoIntValues_ReturnSum(int num1, int num2, int expected)
        {
            var actual = SumOfTwoNumbers.Sum(num1, num2);

            Assert.Equal(expected, actual);
        }
    }
}